firestore-shorten-urls-dynamic-links/functions/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "firestore-shorten-urls-dynamic-links",
+  "scripts": {
+    "lint": "tslint --project tsconfig.json",
+    "clean": "rimraf lib",
+    "build": "npm run clean && tsc"
+  },
+  "main": "lib/index.js",
+  "dependencies": {
+    "firebase-admin": "^8.6.0",
+    "firebase-functions": "^3.9.1",
+    "node-fetch": "^2.6.0"
+  },
+  "devDependencies": {
+    "@types/node": "^10.17.28",
+    "@types/node-fetch": "^2.5.7",
+    "rimraf": "^3.0.2",
+    "tslint": "^5.12.0",
+    "typescript": "^3.8.0"
+  },
+  "engines": {
```

Comment:
Please remove "engines" field, it gets ignored.

Reply:
"engines" field is used by CF3 to determine runtime, but doesn't do anything for extensions functions.